cdap-docs/admin-manual/source/upgrading/packages.rst

```diff
@@ -47,19 +47,13 @@ upgrade instructions for the earlier versions and upgrade first to
 
 #. Update the CDAP packages by running either of these methods:
 
-   - On RPM using Yum (on one line)::
+   - On RPM using Yum::
 
-       $ sudo yum install cdap cdap-gateway \
-             cdap-hbase-compat-0.96 cdap-hbase-compat-0.98 cdap-hbase-compat-1.0 \
-             cdap-hbase-compat-1.0-cdh cdap-hbase-compat-1.1 \
-             cdap-kafka cdap-master cdap-security cdap-ui
+       $ sudo yum install `rpm -qa --qf "%{NAME}\n" 'cdap*'`
```

Comment:
no, it was only for consistency with what's already there.  I think the reason we originally had to list the packages was when we swapped cdap-web-app for cdap-ui.  But Chris's suggestion is fine.  I think the simplicity outweighs the chance to pull in extra packages.  And if we ever swap packages again we can modify for that release.  One correction: ensure quotes around ``'cdap*'``:

``sudo yum upgrade 'cdap*'``
``apt-get install --only-upgrade 'cdap*'``